config/config.go

```diff
@@ -52,6 +52,10 @@ type Configuration struct {
 	UploadedImagesDir string `default:"data/images"`
 	PluginsDir        string `default:"data/plugins"`
 	Registration      bool   `default:"false"`
+	PriorityColors    []struct {
```

Comment:
I don't think this needs to be configurable. We can hardcode the colors. i think something like this as border color should be enough:
```
rgba(243, 156, 18, Math.min(1, msg.priority/10))
```